### DISCUSSION TITLE
Poll: New `UpdateBeatmapSetButton` design

### DISCUSSION THREAD

**Discussion by normalid-awa** [ppy/osu#29062] [Polls]
The old one looks more like a state's indicating text instead of button. When I first met this "button" I thought it was just indicating the map is updating as it does not appear as a button. And I figure it out by the time that i accidentally hover on it.

Therefore to eliminate the confusion, I turn the background to more like a button belike.

| | New design | Old design |
|--------|--------|--------|
| Normal | ![image](https://github.com/user-attachments/assets/4888d468-b3f4-4b49-a1f8-316dd49e4ddc) | ![image](https://github.com/user-attachments/assets/c2b3f2a1-443a-425e-8530-d3bb27ab44ab) |
| Hover | ![image](https://github.com/user-attachments/assets/6f7d32f7-2059-473e-9a26-2751f567d752) | ![image](https://github.com/user-attachments/assets/bbdae61b-6763-4e46-809e-f43c0f63adfa) |
| Updating| ![image](https://github.com/user-attachments/assets/7a3283d5-d5a1-4b05-9884-78138d59ad1f)| ![image](https://github.com/user-attachments/assets/a846821c-a20f-4189-8ec7-e438a1466c0a) |

**Comment by bdach** [comment]
psst: https://github.com/ppy/osu/issues/22204

**Reply by uzervlad** [reply]
But it's not done yet, is it?

No harm in making small tweaks like this while the new design is still WIP.

**Reply by bdach** [reply]
the "new tweak" is dated already. we don't do the "filled triangles" anywhere anymore. going in that direction is wasted energy.
